lesson26/src/js/login.js

```diff
@@ -87,3 +89,8 @@ for (const input of inputSelector){
 }
 
 togglePasswordButton.addEventListener('click', () => togglePassword(passwordInput,togglePasswordButton));
+
+const isRelatedTogglePassword = (event) => {
+    const related = event.relatedTarget ? event.relatedTarget.id : "unknown";
+    return related === "js-toggle-password";
+}
```

Comment:
If you try to get the `event.relatedTarget.id` directly, it will not work, so this is how it is written.
 ref: https://developer.mozilla.org/ja/docs/Web/API/MouseEvent/relatedTarget#javascript